docs/ref/forms/widgets.txt

```diff
@@ -286,6 +286,23 @@ foundation for custom widgets.
         an HTML form submission, so it's unknown whether or not the user
         actually submitted a value.
 
+    .. method:: use_required_attribute(initial)
+
+        .. versionadded:: 1.10.1
+
+        Given a form field's ``initial`` value, returns whether or not the
+        widget should be rendered with the ``required`` HTML attribute.
```

Comment:
I'm thinking "should -> can" or something so that it's clear that this method doesn't account for `field.required`.